libs/sdk-core/src/swap.rs

```diff
@@ -128,7 +129,10 @@ impl BTCReceiveSwap {
     /// Create a [SwapInfo] that represents the details of an on-going swap.
     ///
     /// See [SwapInfo] for details.
-    pub(crate) async fn create_swap_address(&self) -> Result<SwapInfo> {
+    pub(crate) async fn create_swap_address(
+        &self,
+        channel_opening_fees: OpeningFeeParams,
+    ) -> Result<SwapInfo> {
```

Comment:
There is a problematic case when the user created a swap address with some fees and now the user invoke this API to create a swap address and get the same reused one (see bellow  `self.list_unused()` )
In this case the user gets an unused one from the db but it is attached to the old fee while now there are new fees that the user has chosen (and saw).

We may need to update the fees here for this address.

Reply:
Done: d4fb0e11c76fac630f15abbbe400a4732d138ed9